### DISCUSSION TITLE
امنیت خروجی ها

### DISCUSSION THREAD

**Discussion by ghost** [hiddify/Hiddify-Manager#211] [Ideas ایدهها]
@hiddify  خسته نباشی
اخیرا موردی پیش میومد برای دوستان می گفتن پردازش در حد 100% میشه تو پنل کاشف به عمل اومد کانکشن هایی که ایجاد می کنن خیلی سریع پخش میشه  بیشتر این کسایی هم که متصل بودن چینی بودن اونم از ایونتهای کلود دیدم (کلی هم اتک)
 
چند تا رول نوشتم و کمی بهبود پیدا کرد(آی پی های خارج از کشور قطع شدن)
آیا امکان این هست که هاردویر آی دی برای هر یوزر موقع کرییت کردن یوزر بشه برای دستگاه همون یوزر ادد کرد 

و یه خواهش دیگه ایجاد پورتها دیگه
https : 2053 2083 2087 209 68443

**Comment by hiddify-com** [comment]
وایسا 
اتک به کجا میشده؟ جزییات بیشتر میدی؟
از کدوم راه استفاده می کردند برای نفوذ؟

**Reply by ghost** [reply]
![Untitled](https://user-images.githubusercontent.com/125456733/221222001-57ec9d3e-7170-4ae5-b746-507b05892964.jpg)
به اون طرف دسترسی ندرام ولی  اون کانشکشن هاش پخش شده بود  چون تو پنل x-ui هم مورد که داشت چون تو چنل تلگرام می ذاشتش آثاری از هک وجود نداشت, اینم 
اسکرین شات مال خودم طی 24 ساعت گذشته هست بعضی پروتوکل ها که می بستم بازم می کردم کم و زیاد می شدن مورد ها بلاک شده 
از tor هم داشتم
گفتن ختلال تو آلمان زیاد هست اومدم کشور دیگه این وضعیت بهتر شده ام هست

**Comment by hiddify-com** [comment]
منظورم اینه از چه تکنیکی استفاده میکنند؟
میتونی دسترسی ssh به سرورت بهم بدی چک کنم؟
hiddify@gmail.com

**Reply by ghost** [reply]
وااله سواتم اونقده نیست D:
ایجنت هاشون رو چک میکردم بعضیاشون بات بودن
چشم
